spec/rails_helper.rb

```diff
@@ -63,6 +66,18 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.around(:each, type: :request) do |example|
+    host! ENV['CLAIMS_HOST']
+    example.run
+    host! nil
+  end
+
+  config.around(:each, type: :system) do |example|
+    Capybara.app_host = "http://#{ENV['CLAIMS_HOST']}"
+    example.run
+    Capybara.app_host = nil
+  end
```

Comment:
It looks like there's duplication here between this config and the config manually applied on system specs in @CatalinVoineag's PR.

https://github.com/DFE-Digital/itt-mentor-services/blob/1f7ff126fafb8160301460e2fba88a2d49720da6/spec/system/claims/view_organisations_spec.rb#L4-L8

Also: how will this config know which hostname to use – `CLAIMS_HOST` or `PLACEMENTS_HOST`?

Reply:
This is the default, it sets the host to `CLAIMS` by default. I've made the change as I saw a previous PR sets `CLAIMS` to be the default host.

For system tests, I've added the `given_i_am_on_the_xxxxx_site` helper method, which is the only time we should really need to think about the site we're on. In unit tests, we shouldn't need to think about it.

Alternatively, we _could_ pass in a `service: :claims` or `service: :placements` in the spec describe options,

```ruby
RSpec.describe "Home Page", type: :system, service: :claims do
```